lib/index.ts

```diff
@@ -189,12 +197,12 @@ export class DashLine {
         this.cursor.set(x, y)
         return this
     }
-
+ 
```

Comment:
Because pixi v8 requires you to adjust line styles after drawing the paths, you'll need a call to `this.setStrokeStyle()` here at the end of this method.